tests/serverpod_test_server/test_integration/test_tools/start_timeout_test/start_timeout_test.dart

```diff
@@ -37,15 +32,11 @@ void main() async {
           contains(
               'Serverpod did not start within the timeout of 0:00:30.000000'));
       expect(
-          timer.elapsed.inSeconds,
-          allOf(
-            greaterThanOrEqualTo(30),
-            lessThan(40), // sanity check
-          ));
+        timer.elapsed.inSeconds,
+        greaterThanOrEqualTo(30),
```

Comment:
Question: Are we actually waiting 30 seconds in this test?

Reply:
Yes, when not specifying a timeout, it seems we are hardcoded to use 30s:

The test checks how long it takes until `withServerpod`  fails, when not specifying a timeout, and no database exists to be found.

from `with_serverpod.dart:80`
```dart
var startTimeout = maybeServerpodStartTimeout ?? const Duration(seconds: 30);
```

Smells a bit, but that is what is happening today.